cannon/example/claim/go.mod

```diff
@@ -7,7 +7,7 @@ toolchain go1.21.1
 require github.com/ethereum-optimism/optimism v0.0.0
 
 require (
-	golang.org/x/crypto v0.18.0 // indirect
+	golang.org/x/crypto v0.19.0 // indirect
```

Comment:
Was this intended to be committed here?

Reply:
Yes, the build fails without it.  Seems to be broken on develop again somehow.